pages/embedded-portal.tsx

```diff
@@ -292,38 +292,43 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
 
   console.log("spaceData", spaceData);
 
+  console.log("getWorkbook race condition error occuring below. Resolving...");
```

Comment:
I can prevent the workbook error console log by wrapping async with the pattern below, but I would have to add a bunch of conditionals to the code

Reply:
undid